### PR TITLE
Implement canceled event flagging and display

### PR DESCRIPTION
**Pull request by jsturek**
To test use: https://events-dev.unl.edu

Need to determine how to display canceled events to via HTML, RSS, XML, JSON, and ICAL formats.

### CODE REVIEW

**Comment by jsturek** [comment]
@skoolbus39 and @ericras This PR is now ready for review and may be tested on https://events-dev.unl.edu.

File: src/UNL/UCBCN/Event.php [hunk 1]
```diff
@@ -624,4 +625,16 @@ public function userCanEdit($user = NULL)
...
+    }
+
+    public function displayTitle() {
+        return $this->isCanceled() ? 'Canceled: ' . $this->title : $this->title;
+    }
+
+    public function icalStatus() {
+        return $this->isCanceled() ? 'CANCELLED' : 'CONFIRMED';
+    }
 }
```
Comment (skoolbus39) [inline src/UNL/UCBCN/Event.php:640]: No newline at end of file

**Review by skoolbus39 — CHANGES_REQUESTED** [review]

**Review by skoolbus39 — APPROVED** [review]

**Review by ericras — COMMENTED** [review]
"status" isn't a valid RSS element: https://validator.w3.org/feed/docs/rss2.html

I'd probably just change it to "category" - that might prohibit using "category" in the future for actual categories but maybe not. "Canceled" can always be a category kind of like a tag.

**Comment by jsturek** [comment]
@ericras  I moved the cancel status to the description.  I don't think using category would be utilizing it as intended.

**Review by ericras — APPROVED** [review]

**Pull request merged**